src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -333,7 +333,9 @@ Copyright (c) .NET Foundation. All rights reserved.
     </Otherwise>
   </Choose>
 
-  <PropertyGroup Condition="'$(RoslynCompilerType)' == 'Core'">
+  <!-- If Microsoft.Net.Compilers.Toolset package is referenced (which we can tell by RoslynTasksAssembly being non-empty),
+    don't overwrite target paths - the toolset package should always take precedence. -->
+  <PropertyGroup Condition="'$(RoslynCompilerType)' == 'Core' and '$(RoslynTasksAssembly)' == ''">
```

Comment:
In this new world though do we still want to keep preferring this package? Instead should we consider simplifying by saying that we only support Microsoft.Net.Sdk.Compilers.Toolset? Customers who previously used MS.Net.Compilers.Toolset can use the SDK version and then just control the version?

Reply:
The Sdk toolset is built from the sdk repo though, so we couldn't quickly create a toolset package (e.g., to ship hotfixes) from roslyn, we would need to insert to sdk first (with one more hop through vmr nowadays).

Also the Sdk toolset is framework only I think.